src/PrestaShopBundle/Resources/config/services/adapter/kpi.yml

```diff
@@ -54,3 +54,31 @@ services:
             - '@translator'
             - '@prestashop.adapter.legacy.kpi_configuration'
             - '@=service("prestashop.adapter.legacy.context").getAdminLink("AdminStats", true, {"ajax": 1, "action": "getKpi", "kpi": "products_per_category"})'
+
+    prestashop.adapter.kpi.most_common_customers_gender:
+        class: 'PrestaShop\PrestaShop\Adapter\Kpi\MostCommonCustomersGenderKpi'
+        arguments:
```

Comment:
since all these classes have the same constructor maybe you could create an abstract class with a default service definition, and make the other definitions inherit it

Reply:
maybe, but in another PR, there are quite more KPI classes.